src/Entity/EntityBrowser.php

```diff
@@ -26,8 +26,18 @@
  *   handlers = {
  *     "form" = {
  *       "entity_browser" = "Drupal\entity_browser\Form\EntityBrowserForm"
+ *     },
+ *     "access" = "Drupal\entity_browser\EntityBrowserAccessControlHandler",
+ *     "list_builder" = "Drupal\entity_browser\Controllers\EntityBrowserListBuilder",
+ *     "wizard" = {
+ *       "add" = "Drupal\entity_browser\Wizard\EntityBrowserWizardAdd",
+ *       "edit" = "Drupal\entity_browser\Wizard\EntityBrowserWizard"
  *     }
  *   },
+ *   links = {
+ *     "canonical" = "/admin/config/content/entity_browser/{entity_browser}",
```

Comment:
"edit-form" needs to be added too as list builder depends on it. I suspect the same with "delete-form" (which we'll need to add in this case).